CONTRIBUTING.md

```diff
@@ -49,6 +49,7 @@ Builders/Constructors
 * For interfaces that are meant to be extended by the user but that also have a builtin implementation, you can use the `Default${Iface}` naming pattern (see DefaultUpload)
 * If there are several builtin implementations, use a descriptive name (like AppSyncWsProtocol, ...)
 * Avoid top level constructor functions like `fun CoroutineScope(){}` because they are awkward to use in Java
+* For expect/actual, it's sometime convenient to expose an interface even if it's not intended to be used by callers like `MockerServerIface`. In that case, it's ok to use `FooIface` for the interface and `Foo()` for the implementation to avoid having "DefaultFoo" everywhere when there's only one "Foo". 
```

Comment:
Very subjective: I'm not very fond of the `Iface` suffix and would prefer `Interface` (or even `I` prefix)...  But it's really not a big deal 😅

Reply:
Agreed, rename here: https://github.com/apollographql/apollo-android/pull/3617/commits/c8795b94a39d10f1e9a3c218acfbe6ef79ddbd94